extensions/azurecore/src/account-provider/tokenCache.ts

```diff
@@ -32,12 +32,32 @@ export default class TokenCache implements adal.TokenCache {
 				.then(cache => self.addToCache(cache, entries))
 				.then(updatedCache => self.writeCache(updatedCache))
 				.then(
-					() => callback(null, false),
-					(err) => callback(err, true)
+					() => callback(null, true),
```

Comment:
Oh and this was a bug I noticed while fixing the other issue where we were passing the wrong result value back